provider/dynamodb.go

```diff
@@ -113,17 +114,17 @@ func CreateMetadataTable(dynamodbClient *dynamodb.DynamoDB) error {
 	}
 	_, err := dynamodbClient.DescribeTable(describeMetadataTableParams)
 	if err != nil {
-		fmt.Println("Could not describe dynamo metadata table, attemping to create...", err)
+		fmt.Println("Could not describe dynamo metadata table, attempting to create...", err)
```

Comment:
Do we have the log library in here?

Reply:
We do not, and actually many of the other provider do not either (e.g. Redis). I'm happy to add logging here, though.

Reply:
Logging added to DynamoDB.